src/main.js

```diff
@@ -149,7 +161,8 @@ module.exports = {
             const ret = {
               filePath,
               type: severity === 1 ? 'Warning' : 'Error',
-              range
+              range,
+              fix: linterFix
```

Comment:
(It works in Nuclide since we check for presence via `message.fix != null`, but it should probably get left off altogether if there is no fix)